docs/user/ppl/functions/condition.rst

```diff
@@ -192,3 +192,49 @@ Example::
     | False  | Nanette   | Bates    |
     | False  | Dale      | Adams    |
     +--------+-----------+----------+
+
+CASE
+------
+
+Description
+>>>>>>>>>>>
+
+Usage: case(condition1, expr1, condition2, expr2, ... conditionN, exprN else default) return expr1 if condition1 is true, or return expr2 if condition2 is true, ... if no condition is true, then return the value of ELSE clause. If the ELSE clause is not defined, it returns NULL.
+
+Argument type: all the supported data type, (NOTE : there is no comma before "else")
+
+Return type: any
+
+Example::
+
+    os> source=accounts | eval result = case(age > 35, firstname, age < 30, lastname else employer) | fields result, firstname, lastname
+    fetched rows / total rows = 4/4
+    +--------+-----------+----------+
+    | result | firstname | lastname |
```

Comment:
add age and employer, `fields result, age, firstname, lastname, employer`

Reply:
done